relint/parse.py

```diff
@@ -140,13 +144,17 @@ def print_culprits(matches, args):
 
     if args.summarize:
         for test, filenames in match_groups.items():
-            hint = Panel(
-                Markdown(test.hint, justify="left"),
-                title="Hint:",
-                title_align="left",
-                padding=(0, 2),
-            )
-            group = Group(Group(*filenames), hint)
+            if test.hint is not None:
```

Comment:
You should avoid negations in if-else statements to improve readability. Just flip them and drop the negation.
```suggestion
            if test.hint:
```

Reply:
Done.